docker/etc/ros_entrypoint.sh

```diff
@@ -22,7 +22,7 @@ else
     # Add sudo privileges to the user
     echo "$USER_NAME ALL=(ALL) NOPASSWD:ALL" >>/etc/sudoers
 
-    # Source ROS2
+    # Source ROS 2
```

Comment:
Please focus on the minimum change. If you want to refactor please make another PR.
```suggestion
    # Source ROS2
```